### PR TITLE
Dedup IsFileSectorAligned() to fix unity build.

### PR DESCRIPTION
**Pull request by siying**
Summary:
Unity build fails because of name conflict of IsFileSectorAligned() after recent refactoring. Consolidate the function.

Test Plan: make unity. At least the failure goes away. Also "make all", "make release" and see no regression in normal cases.

### CODE REVIEW

File: file/read_write_util.h [hunk 1]
```diff
@@ -26,4 +26,9 @@ extern Status NewWritableFile(Env* env, const std::string& fname,
 bool ReadOneLine(std::istringstream* iss, SequentialFile* seq_file,
                  std::string* output, bool* has_data, Status* result);
 
+#ifndef NDEBUG
+bool IsFileSectorAligned(const size_t off, size_t sector_size) {
```
Comment (ltamasi) [inline file/read_write_util.h]: I think we should mark this inline if we're going to put the implementation in the header file (alternatively, the definition could be moved to the .cc).

File: file/readahead_raf.cc [hunk 2]
```diff
@@ -15,15 +15,6 @@
 #include "util/rate_limiter.h"
 
 namespace rocksdb {
-
```
Comment (ltamasi) [inline file/readahead_raf.cc]: `#include "file/read_write_util.h"` in this file as well?

File: file/read_write_util.h (hunk 1 above)
Comment (siying) [reply file/read_write_util.h]: Yes, I noticed it too. I moved it to .cc file now.

File: file/readahead_raf.cc (hunk 2 above)
Comment (siying) [reply file/readahead_raf.cc]: Fixed. Good catch.

**Pull request closed without merging**